src/MSALWrapper/Constants.cs

```diff
@@ -20,6 +20,11 @@ public static class Constants
         /// </summary>
         public static readonly Uri AadOAuthRedirectUri = new Uri("urn:ietf:wg:oauth:2.0:oob");
 
+        /// <summary>

```

Comment:
Given the main usage of this value is in the CLI project, this isn't actually a constant inherent to the MSALWrapper - I think we can define this on the CommandMain class itself.